scripts/pre_commit_linter.py

```diff
@@ -866,7 +868,8 @@ def _check_newline_character(all_files):
     summary_messages = []
     all_files = [
         filename for filename in all_files if not
-        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)]
+        any(fnmatch.fnmatch(filename, pattern)
+            for pattern in EXCLUDED_PATHS_EXT)]
```

Comment:
Instead of creating a new variable for this, can you please accommodate `.py` extension here directly?

Reply:
Done!